css-text-4/Overview.bs

```diff
@@ -4603,8 +4456,35 @@ Breaking Rules for Letters: the 'word-break' property</h3>
 			(which uses [=spaces=] between words),
 			and is also useful for mixed-script text where CJK snippets are mixed
 			into another language that uses [=spaces=] for separation.
+
+		<dt><dfn>auto</dfn>
+		<dd>
+			This value directs the user agent to perform language-specific content analysis
```

Comment:
How about making the name more explicit? e.g., naming the value `phrase` and define it "to prioritize keeping natural phrases (of multiple words) together" (borrowed from the [text in LDML](https://www.unicode.org/reports/tr35/#:~:text=Prioritize%20keeping%20natural%20phrases%20(of%20multiple%20words)%20together%20when%20breaking%2C%20used%20in%20short%20text%20like%20title%20and%20headline))? Other value name is fine too, I just wonder if it's better to be more explicit.

In Thai, both word-boundary and phrase-boundary require language-specific content analysis. Today, by default, ICU uses dictionary-based (may switch to ML-based in future) content analysis to determine word boundaries. This behavior is for `word-break: normal`, not for this new value.

On the other hand, there are requests for [BudouX](https://github.com/google/budoux) to support Thai to break lines only at natural phrase (of multiple words) boundaries, not at every word boundary.